### DISCUSSION TITLE
Tablet - Schedule View

### DISCUSSION THREAD

**Discussion by VincentAudette** [ApplETS/Notre-Dame#362] [Ideas] [answered]
Liste sur le côté du calendrier. Cette liste pourrait être scrollable et selon les items dans le scroll view la date du calendrier change. Également si tu change la date pour une case du calendrier le scroll débuterait avec la première activité de cette journée.

<img width="1428" alt="Screen Shot 2022-01-16 at 11 29 08 PM" src="https://user-images.githubusercontent.com/36488653/149709366-eb24ca17-6bd6-4075-a322-77e5954a9a21.png">

**Comment by apomalyn** [comment, marked as answer]
Will be related to #180 !! 😄

**Comment by VincentAudette** [comment]
@MysticFragilist fait référence à #211 également.
